uber/model_checks.py

```diff
@@ -448,8 +449,19 @@ def valid_uses_allowed(promo_code):
 
 
 @validation.PromoCode
+def no_unlimited_free_badges(promo_code):
+    if promo_code.is_unlimited and (promo_code.is_free or (
+            promo_code.discount_type == PromoCode.FIXED_DISCOUNT and
+            promo_code.discount >= c.BADGE_PRICE)):
+        return 'Unlimited-use, free-badge promo codes are not allowed.'
```

Comment:
Yeah, keep in mind there's percent discounts and fixed-price discounts, which don't have this issue, although at this point I realize that my 'minimum price' column (which was refactored out in favor of discount type) would be the ideal solution there. That being said, I kinda doubt we need it -- admins have lots of options, and also why would you make a promo code of minus $50 \D:/